examples/step-54/doc/intro.dox

```diff
@@ -66,8 +66,8 @@ the OpenCASCADE framework. From a <code>TopoDS_Shape</code>, it is then
 possible to access all the sub-shapes (such as vertices, edges and faces)
 composing it, along with their geometrical description.  In the deal.II
 framework, the topological entities composing a shape are used to create
-objects of the Manifold or Boundary classes. In Step-6 we saw how to build a
```

Comment:
I read the sentence

> the topological entities composing a shape are used to create objects of the Manifold classes.

several times and I honestly have no idea what this means. Could we reword this sentence?

Reply:
I only changed a few words. Does this sound better to you?

Reply:
I think so, but I honestly still don't understand it. What does 'topological entities' mean here?

Reply:
"topological entities" is what OpenCASCADE uses to define an abstract geometrical object, or a shape (a `Topo_DS_Shape`). Shapes can be vertices, edges, faces, volumes, and any combination of the above. We have no such a thing in deal.II, but `OpenCASCADE` does, and this documentation refers somehow to this nomenclature.

Reply:
In OpenCASCADE a `TopoDSShape` is a collection of topological shapes. Some of our manifolds work with faces, other with edges, but all of them can be encapsulated in a OpenCASCADE `Shape`, which is the entry that is returned, for example, when opening IGES files.

Reply:
Yes, this makes sense now. Thanks.